metafile-schema.json

```
@@ -57,7 +57,7 @@
              "examples": [
                "data.csv"
              ],
-             "pattern": "^(.*)(\\.(csv|txt)(\\.gz)?(\\.gpg)?|\\.zip)$"
+             "pattern": "^(.*)(\\.(csv|txt|Tab|tab)(\\.gz)?(\\.gpg)?|\\.zip)$"
```

```suggestion
             "pattern": "^(?i)(.*)(\\.(csv|txt|Tab|tab)(\\.gz)?(\\.gpg)?|\\.zip)$"
```
Aren't we making this change, then?

---

I'm just insisting on it because if KXS has `.Tab` files then they probably also have `.TXT` and `.Csv` and whatnot 😛

---

oh, just need lowercase `tab`?

---

The `(?i)` I'm adding to the front of it should make the _whole_ regex case-insensitive.

---

got it. thanks

---

done in https://github.com/rubikloud/metafile-json-schema/pull/2/commits/3832b524e938eca345be684625d7359bd7881d13